core/examples/luigi-sample-angular/webpack.config.js

```diff
@@ -397,6 +397,27 @@ module.exports = {
           from: {
             glob: 'luigi-client.js'
           }
+        },
```

Comment:
The css file is copied, but not injected into index.html
A quick tryout of `webpack.config.js`  entry.styles: ['./src/styles.css', './node_modules/fundamental-ui/dist/fundamental-ui.min.css']   did not work in my case, needs investigation how to reference the css properly.

Reply:
Alternatively manually put it into sampleapp.html and remove the injection reference in angular.json

Reply:
`fundamental-ui.min.css` injected via `webpack.config.js`. Please check again.

Reply:
Good job!